src/utils.test.ts

```diff
@@ -0,0 +1,19 @@
+import {decimalKeypad, notEmpty} from './utils';
+
+test('utility function notEmpty', () => {
+  const testCase = ['', null, undefined, 'Test'];
+  const expected = [false, false, false, true];
```

Comment:
Instead of this you can use [each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) to make tests easier to debug. If the test fails for a specific input you will know which one exactly